packages/calcite-components/src/components/tab/tab.scss

```diff
@@ -18,17 +18,16 @@ section,
   @apply hidden h-full w-full;
 }
 
-:host([scale="s"]) {
+.scale-s section {
```

Comment:
Not needed for this PR, but we can make these styles more robust by using a class vs an element.